apps/admin/backend/src/cast_vote_records.ts

```diff
@@ -47,7 +47,6 @@ import {
   CvrFileMode,
   ImportCastVoteRecordsError,
 } from './types';
-import { buildGetUsbDrivesFn } from './util/exporter';
```

Comment:
Can `buildGetUsbDrivesFn` be removed completely now?

Reply:
I'm not sure if it can be removed in this PR, but it is removed in one of the PRs in this series.